### PR TITLE
Editable list

### PR DESCRIPTION
**Pull request by alef-garrido**
Project Requirements: 

- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for editing task descriptions.
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for the "Clear all completed" button (use filter() method).
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
+/* eslint-disable radix */
```
Comment (oxenprogrammer) [inline src/index.js:4]: **Optional Changes**
- [ ] The linters are there to help follow some best practices. But sometimes I understand they are much of a pain and get into our ways. I will advice that you try as much as possible to work with some of them like strict-mode and no-globals where possible. Using globals can easily introduce leakage to your application.

Comment (oxenprogrammer) [inline src/index.js:4]: **Optional Changes**
- [ ] The linters are there to help follow some best practices. But sometimes I understand they are much of a pain and get into our ways. I will advice that you try as much as possible to work with some of them like strict-mode and no-globals where possible. Using globals can easily introduce leakage to your application.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @alef-garrido , 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
You have aced all the technical requirements of this project except the README file.
It's the only issue that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:

## Required Changes ♻️
- [ ] The README file has a broken image and the link to the live demo is outdated as well. The README file acts as a documentation for your project. Let us try to update it every time we introduce a changelog.
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (alef-garrido) [reply src/index.js:4]: Point taken! Thank you for your time. I did fix some of them but these have been failing still, but be sure i'll get it right.

**Comment by alef-garrido** [comment]
Please consider i have remove the link for the live-Demo as Git-hub pages wont display distribution folder bundle's. I have place a note advising to run a clone of the repo using the live-server provided in the dependencies of this project.

**Review by oxenprogrammer — APPROVED** [review]
Hi @alef-garrido , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alef-garrido** [comment]
Hey @oxenprogrammer the live demo is now working and i have added the link to the README file. Thank you for your time!

**Pull request merged**